k8s/deployment.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: promotions
+  labels:
+    app: promotions
+spec:
+  replicas: 2
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 0%
+      maxUnavailable: 50%
+  selector:
+    matchLabels:
+      app: promotions
+  template:
+    metadata:
+      labels:
+        app: promotions
+    spec:
+      restartPolicy: Always
+      containers:
+        - name: promotions
+          image: cluster-registry:32000/promotions:latest
```

Comment:
This image should be the same used in `make cluster` command. In `Makefile`, we use `cluster-registry:5000`.

Reply:
Gotcha! Lemme change it to the current image.